setup.py

```diff
@@ -10,15 +10,16 @@
     long_description = f.read()
 
 setup(
-    name="fee_lab_to_nwb",
+    name="fee-lab-to-nwb",
```

Comment:
@weiglszonja So what this change necessary or just the one below?

Reply:
No, it even works with "fee_lab-to_nwb", would install the package as 
```... Successfully installed fee-lab-to-nwb-0.0.1```
I can rollback this change, or keep it to be consistent with the other packages?

Reply:
Yeah I guess we do usually use the `-`'s